scripts/linters/general_purpose_linter_test.py

```diff
@@ -196,32 +196,6 @@ def test_invalid_use_of_browser_wait_for_angular(self):
         self.assertEqual('Bad pattern', lint_task_report.name)
         self.assertTrue(lint_task_report.failed)
 
-    def test_invalid_use_of_fdescribe_ddescribe(self):
-        linter = general_purpose_linter.GeneralPurposeLinter(
-            [INVALID_FDESCRIBE_DDESCRIBE_FILEPATH], FILE_CACHE)
```

Comment:
Please delete the corresponding test files*

Reply:
Done